stdlib/public/Platform/CMakeLists.txt

```diff
@@ -473,6 +473,11 @@ if("ANDROID" IN_LIST SWIFT_SDKS)
                                  COMPONENT sdk-overlay)
     endif()
   endforeach()
+
+  swift_install_in_component(FILES
+      posix_filesystem.apinotes
+    DESTINATION "share"
```

Comment:
Do you install the compiler and this file on Windows before building `swift-corelibs-foundation` with that new installed location? I just tried using this with #5010 on Android and  this file wasn't available, because `build-script` uses the freshly-built compiler in `build/` to compile `swift-corelibs-foundation`, not the installed one from `--install-prefix`.

I will submit a change to place this file in `build/` also, so we can use it when cross-compiling the corelibs for Android with `build-script` also.

Reply:
Yes, we install the compiler first to use it for the subsequent steps. I think that it makes more sense to change build-script to alter the behaviour and to stage the compiler first.